tinker/admin/program_search/__init__.py

```diff
@@ -66,22 +66,28 @@ def submit(self):
 
     @route('/multi-delete', methods=['POST'])
     def multi_delete(self):
-        ids_to_delete = json.loads(request.data)
-        for id in ids_to_delete:
-            if not (isinstance(id, str) or isinstance(id, unicode)):
+        list_of_ids_to_delete = json.loads(request.data)
+        for id_to_delete in list_of_ids_to_delete:
+            if isinstance(id_to_delete, unicode):
+                id_to_delete = id_to_delete.encode('utf-8').strip()
+
+            if isinstance(id_to_delete, str):
+                ProgramTag.query.filter_by(id=id_to_delete).delete()
+            else:
                 return "One of the ids given to this method was not a string"
-            ProgramTag.query.filter_by(id=id).delete()
         db.session.commit()
         self.base.create_new_csv_file()
-        return 'Deleted ids: ' + ', '.join(ids_to_delete)
+        return 'Deleted ids: ' + ', '.join(list_of_ids_to_delete)
 
     @route('/search', methods=['post'])
     def search(self):
         try:
-            data = json.loads(request.data)
+            data = EncodingDict(json.loads(request.data))
```

Comment:
This is excessive I think. We should only have to do this with things syncing to Cascade

Reply:
I considered that, but I figured that we should sanitize all POST data in this one pull request instead of only sanitizing things as they happen to break.